lib/node_modules/@stdlib/math/base/complex/divide/lib/index.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+/**
+* Divide two complex numbers.
+*
+* @module @stdlib/math/base/complex/divide
+*
+* @example
+* var cdiv = require( '@stdlib/math/base/complex/divide' );
+*
+* var v = cdiv( -13.0, -1.0, -2.0, 1.0 );
+* // returns [ 5.0, 3.0 ]
+*/
```

Comment:
Small feedback: it seems like the same examples are duplicated in at least 6+ places or so. Some have different purposes, but many are identical. I remember @Planeshifter doing some automation work to insert examples into the readme. Is that a goal? It would certainly save some iteration and copy/pasting.